### PR TITLE
340/Rocket-boop-on-mouse-enter

### PR DESCRIPTION
**Pull request by pierwszazlewej**
- inital attempt to make graphic move :)

### CODE REVIEW

**Comment by pierwszazlewej** [comment]
obraz

Lil rocket is launching! 
Its on the initial screen, ready to move when the desktop is ready :)

File: client/src/Atoms/Rocket/RocketStyled.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const RocketStyled = styled.div`
+  transform: translate(0%);
```
Comment (KatarzynaChmielecka) [inline client/src/Atoms/Rocket/RocketStyled.js:4]: You don't need line 4 and 5.

File: client/src/Atoms/Rocket/RocketStyled.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  transition: 0.3s ease-out;
+
+  &:hover {
+    transform: translate(-5%, -30%);
+    transition: 0.3s ease-out;
```
Comment (KatarzynaChmielecka) [inline client/src/Atoms/Rocket/RocketStyled.js:9]: You should add transform property to transition. We shouldn't animate all things :)

File: client/src/Molecules/RocketWithPlanets/RocketWithPlanets.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import Rocket from '../../Atoms/Rocket/Rocket';
+import RocketWithPlanetsStyled from './RocketWithPlanetsStyled';
+
+const RocketWithPlanets = () => {
+  return (
+    <RocketWithPlanetsStyled>
+      <Rocket></Rocket>
```
Comment (KatarzynaChmielecka) [inline client/src/Molecules/RocketWithPlanets/RocketWithPlanets.js:7]: Rocket component can be self-closing :)

File: client/src/Atoms/Rocket/RocketStyled.js (hunk 1 above)
Comment (pierwszazlewej) [reply client/src/Atoms/Rocket/RocketStyled.js]: I need those. :) They provide a smooth animation on mouse leave

File: client/src/Molecules/RocketWithPlanets/RocketWithPlanets.js (hunk 3 above)
Comment (pierwszazlewej) [reply client/src/Molecules/RocketWithPlanets/RocketWithPlanets.js:7]: done!

**Pull request closed without merging**